Add Project Loved: Week of November 4th news post (#1816)
ppy/osu-wiki: news/2018-11-04-project-loved-week-of-november-4th.md

---
layout: post
title: "Project Loved: Week of November 4th"
date: 2018-11-04 19:00:00 +0000
---

[Project Loved](https://osu.ppy.sh/community/forums/120) returns with another set of beatmaps nominated for the Loved category! Check out each one, and vote for your favourites!

[![](/wiki/shared/news/banners/project-loved.jpg)](https://osu.ppy.sh/community/forums/120)

Welcome back to [Project Loved](https://osu.ppy.sh/community/forums/120)! Every two weeks, we pick out a few beatmaps from each game mode that will get their shot at making it into the [Loved category](https://osu.ppy.sh/beatmapsets?s=8). From the maps we selected, you can now vote whether or not you want them to be Loved! Click on each maps' header image in this post to be taken to its voting thread.

> Continuing the slow, steady decline of the quantity of maps we put up for vote, this week the osu!mania captains have decided to only put up seven maps, instead of the usual eight. There's also one less map for osu!standard, but that was actually the result of a misunderstanding between us and a mapper that didn't want their map to be put up for voting. There will be ten maps again next time.

Author: we are (very slowly) lowering the amount of maps, quantity is correct

it was a misunderstanding cuz we thought they did want their map Loved but turns out not

[Aika - Sakura Trip](https://osu.ppy.sh/beatmapsets/830964#osu) by [CrazyrabbitKGE](https://osu.ppy.sh/users/4481526)  
*written by [BeasttrollMC](https://osu.ppy.sh/users/3171691)*

CrazyrabbitKGE is a name unfamiliar to most, but I promise you it's a name you'll want to remember. His map of "Aika - Sakura Trip" provides a challenging twist for experienced players and sets the canvas for a unique spectator experience with its Sakura flower pattern structure and various keysounded sections. The map garnered over 100 favorites just through word of mouth, so if you want to know what the buzz is about, check out the map and vote for its place in the Loved category!

[![](/wiki/shared/news/2018-11-04-project-loved-week-of-november-4th/osu/speedom-wwc2-feat-krizz-kaliko-and-eminem.jpg)](https://osu.ppy.sh/community/forums/topics/TOPIC_ID)

[Tech N9ne - Speedom (WWC2) (feat. Krizz Kaliko and Eminem)](https://osu.ppy.sh/beatmapsets/397474#osu) by [LukerMaster](https://osu.ppy.sh/users/2093623)  
*written by [BeasttrollMC](https://osu.ppy.sh/users/3171691)*

> Some people may recognize this mapper from another Loved map, "Imaginary Places", which is iconic for its Boss level difficulty. This map is no different. Clocking in over 8 stars, this alt-styled fast aim map by LukerMaster will push you to your limits. Good luck!

Reviewer: ```suggestion
Some people may recognize this mapper from another Loved map, "Imaginary Places", which is iconic for its boss level difficulty. This map is no different. Clocking in over 8 stars, this alt-styled fast aim map by LukerMaster will push you to your limits. Good luck!
```

---
I don't think there is a need to first letter capitalise "boss"

Author: ah I thought it was a diff name, I guess not

*written by [SoraRoxas01](https://osu.ppy.sh/users/1986262)*

This is a great map for people that really want to test their skill with technical maps. "The Government Knows" is a map full of challenging patterns along with both fast and slow sliders which will test how well you can control your aim.  This song is unique in its own way with the constant dance beat in the background and humorous lyrics. You may have recognised Woey previously from his map "Soulless 4" that was a candidate for Project Loved before, along with guest difficulties from Ambrew, MomoHD and Hectique. With the map hitting over 120 favorites, this is a map to definitely give a go if you're up for a challenge!

[![](/wiki/shared/news/2018-11-04-project-loved-week-of-november-4th/osu/concept-of-reality.jpg)](https://osu.ppy.sh/community/forums/topics/TOPIC_ID)

[lapix feat. Renko - Concept Of Reality](https://osu.ppy.sh/beatmapsets/559140#osu) by [ProfessionalBox](https://osu.ppy.sh/users/3250792) et al.  
*written by [SoraRoxas01](https://osu.ppy.sh/users/1986262)*

> One look into this map and you'll know straight away that this is by the unique mapper ProfessionalBox. Not to mention that the song is an absolute banger! But for real though, I'm sure you didn't expect anything less from lapix! Along with ProfessionalBox, there are guest difficulties mapped by other well known mappers like NeilPerry, Sing, Low Combo FC, Haruto and Ameth Rianno. With the eccentric slider shapes and changes in slider velocity during the chorus to go with interesting fillers leading to the build up, this is a map that keeps you toes and is something to have a go at if you're into the more technical and nifty maps!

Reviewer: ```suggestion
One look into this map and you'll know straight away that this is by the unique mapper ProfessionalBox. Not to mention that the song is an absolute banger! But for real though, I'm sure you didn't expect anything less from lapix! Along with ProfessionalBox, there are guest difficulties mapped by other well known mappers like [NeilPerry](https://osu.ppy.sh/users/841391), [Sing](https://osu.ppy.sh/users/3795679), [Low Combo FC](https://osu.ppy.sh/users/3338724), [Haruto](https://osu.ppy.sh/users/3772301) and [Ameth Rianno](https://osu.ppy.sh/users/5219516). With the eccentric slider shapes and changes in slider velocity during the chorus to go with interesting fillers leading to the build up, this is a map that keeps you toes and is something to have a go at if you're into the more technical and nifty maps!
```

I think that the et al mappers should be linked since they're in the description, but it seems weird to do it here.

[S.S.H. - F-zero ~ Big Blue](https://osu.ppy.sh/beatmapsets/154798#osu) by [Saten](https://osu.ppy.sh/users/444506)  
*written by [Bubbleman](https://osu.ppy.sh/users/5182050)*

An iconic song meeting an iconic mapping style, Saten's map of S.S.H.'s Big Blue combines older style patterning with an artist we don't see much of on osu! these days. As such it has a nostalgic feel, but don't let that distract you from the 228 BPM streams and frequent switches between 1/4 and 1/3 rhythms. A lower difficulty has been recently added too, so if you're after a more modern feel with the same Saten style, this is a mapset you don't want to miss—a sentiment shared by at least 60 others.

[![](/wiki/shared/news/2018-11-04-project-loved-week-of-november-4th/osu/stay-for-a-while.jpg)](https://osu.ppy.sh/community/forums/topics/TOPIC_ID)

[Virtual Riot - Stay For A While](https://osu.ppy.sh/beatmapsets/690556#osu) by [ProfessionalBox](https://osu.ppy.sh/users/3250792) and [cRyo[iceeicee]](https://osu.ppy.sh/users/835998)  
*written by [DigitalHypno](https://osu.ppy.sh/users/4384207)*

> Yet another ProfessionalBox map has made its way to the Loved voting stage! This time, it's a control-crazy map of Virtual Riot's "Stay For A While" designed to throw you off with its confusing slider shapes and insanely high slider tick rate of 4. Tied along with quite a catchy and energetic future bass song, this map does an amazing job creating a unique playing experience—I'm sure the 120+ favorites can agree with me on that! So, what say you? Think you've got the cursor control it takes to pass a map like this? If you said yes, I'm going to need you to stop reading this and give the map your best go. Trust me, it'll be worth it! Afterwards, come back to this page and drop a vote on the map's poll!

Reviewer 2: ```suggestion
Yet another ProfessionalBox's beatmap has made its way to the Loved voting stage! This time, it's a control-crazy map of Virtual Riot's "Stay For A While" designed to throw you off with its confusing slider shapes and insanely high slider tick rate of 4. Tied along with quite a catchy and energetic future bass song, this beatmap does an amazing job creating a unique playing experience—I'm sure the 120+ favorites can agree with me on that! So, what say you? Think you've got the cursor control it takes to pass a map like this? If you said yes, I'm going to tell you to stop reading this and give the map your best go. Trust me, it'll be worth it! Afterwards, come back to this page and drop a vote on the beatmap's poll!
```

Author: #osu-wiki and I think "need" is just part of digitalhypno's "go play this!!" style so no change
